### DISCUSSION TITLE
How to setup local database?

### DISCUSSION THREAD

**Discussion by Simplegram** [reworkd/AgentGPT#1011] [Q&A]
I have this error and I don't know what SQL query I should pass to my database.
```
The table `User` does not exist in the current database.
```

**Comment by zulan88** [comment]
进入项目根目录的next目录下，执行
npm install prisma --save-dev
npx prisma db push

**Reply by tongtie** [reply]
```bash
cd next

npm install prisma --save-dev

npx prisma db push
```

**Comment by jasangill1** [comment]
Hello @Simplegram Are you still having problems with that ??
